internal/cli/install.go

```diff
@@ -124,14 +124,76 @@ func (c *InstallCommand) Run(args []string) int {
 	var callOpts []grpc.CallOption
 	tokenResp, err := client.BootstrapToken(ctx, &empty.Empty{})
 	if err != nil {
-		c.ui.Output(
-			"Error getting the initial token: %s\n\n%s",
-			clierrors.Humanize(err),
-			errInstallRunning,
-			terminal.WithErrorStyle(),
-		)
-		return 1
+		// try default context in case server was started again from install
```

Comment:
We should probably only try this logic if `status.Code(err) == codes.PermissionDenied` yes?

My commendation would be something like:

```go
if status.Code(err) != codes.PermissionDenied {
  // The old error message and return 1
}
```

Reply:
Ahh yes, I was curious if we could try this specifically on that error, thanks!